pkg/DESCRIPTION

```diff
@@ -1,10 +1,15 @@
 Package: rhdfs 
 Type: Package
 Title: R and Hadoop Distributed Filesystem
-Version: 1.0.8
-Date: 2013-06-20
+Version: 1.0.9
+Date: 2018-03-05
 Author: Revolution Analytics
-Depends: R (>= 2.6.0), methods, rJava (>= 0.8)
-Maintainer: Revolution Analytics <rhadoop@revolutionanalytics.com>
+Depends: 
+    R (>= 3.3.3),
+    methods,
+    rJava (>= 0.9-9)
+SystemRequirements: Java (>= 1.6)
+Maintainer: Stewart Henderson <stewart.henderson@protonmail.com>
```

Comment:
This repository doesn't appear to be maintained but feel free to call it out and I'll update accordingly.